plugins/plugin-chart-echarts/src/Gauge/buildQuery.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { buildQueryContext, QueryFormData } from '@superset-ui/core';
+
+export default function buildQuery(formData: QueryFormData) {
+  return buildQueryContext(formData, baseQueryObject => [
+    {
+      ...baseQueryObject,
+      groupby: formData.groupby,
+      metrics: [formData.metric, formData.min_val, formData.max_val],
```

Comment:
Is min,max value intended to work as data filter,or just limits on chart?
If used for filter,then I think this wont work.
If only for chart,then it's not required to put it here.

Reply:
I think `min/max` is analogous to [`x_bounds` and `y_bounds`](https://github.com/apache-superset/superset-ui/blob/645539b133e1e02b70fae8002f7a4ad6158d605e/plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx#L230) in other charts, which should just use a `BoundsControl`.

Reply:
You probably don't even need a custom `buildQuery` if you just use a regular `metric` and `bounds` control.